pkg/manager-nnf/manager.go

```diff
@@ -436,12 +436,16 @@ func (s *StorageService) Initialize(log ec.Logger, ctrl NnfControllerInterface)
 
 	s.endpoints = make([]Endpoint, len(conf.RemoteConfig.Servers))
 	for endpointIdx := range s.endpoints {
+		opts := server.ServerControllerOptions{
```

Comment:
This change allows storage groups to be assigned to compute nodes that are down at the time of creation.